modules/sdk-coin-bera/src/lib/walletUtil.ts

```diff
@@ -0,0 +1,601 @@
+export const walletSimpleAbi = [
```

Comment:
Actually I had said to remove this file walletUtil as it is not required, you can address in a followup

Reply:
done